src/Playroom/RenderCode/RenderCode.js

```diff
@@ -1,8 +1,7 @@
-// eslint-disable-next-line import-x/no-unresolved
-import useScope from '__PLAYROOM_ALIAS__USE_SCOPE__';
 import React, { createElement, Fragment } from 'react';
 import scopeEval from 'scope-eval';
 
+import useScope from '../../useScope';
```

Comment:
Turns out the `useScope` entrypoint was never actually used at all, but now it is.